critiquebrainz/frontend/forms/review.py

```diff
@@ -19,14 +19,32 @@ def __call__(self, form, field):
         if l < self.min or self.max != -1 and l > self.max:
             raise ValidationError(self.message)
 
+other_languages_codes = supported_languages
+frequently_used_languages_codes = list(get_top_languages())
+for language in frequently_used_languages_codes:
+    other_languages_codes.remove(language)
```

Comment:
This code leaves this error: 
```
critiquebrainz_1        | Traceback (most recent call last):
critiquebrainz_1        |   File "manage.py", line 14, in <module>
critiquebrainz_1        |     application = DispatcherMiddleware(frontend.create_app(), {
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/__init__.py", line 109, in create_app
critiquebrainz_1        |     from critiquebrainz.frontend.views.review import review_bp
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/views/review.py", line 14, in <module>
critiquebrainz_1        |     from critiquebrainz.frontend.forms.review import ReviewCreateForm, ReviewEditForm, ReviewReportForm
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/forms/review.py", line 27, in <module>
critiquebrainz_1        |     other_languages_codes.remove(language)
critiquebrainz_1        | ValueError: list.remove(x): x not in list
```
This is so because the variable `language` is a `tuple` of language and count. And `frequently_used_language_code` is just a list of languages. Please fix this. :)